docusaurus.config.js

```diff
@@ -60,7 +60,7 @@ const config = {
         title: 'Eclipse Tractus-X',
         logo: {
           alt: 'Eclipse Tractus-X logo',
-          src: 'img/logo.svg',
+          src: 'img/logo_catena.svg',
```

Comment:
This logo is not the catena logo its the tractusx logo.

Please rename it.

Tractus-x is the eclipse project = opensource
catena-x is the consortia = not open source

Catena-x pays you and sponsors the development for tractus-x

Reply:
Hi @Siegfriedk, thanks! the change has been made